scripts/minikube-k8sibmc-all-tests.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n.---------------------------------------\n${1}\n'---------------------------------------\n"
+  set -x
+}
+
+set -ex
+
+# This is one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+
+case ${1} in
+    k8s)
+        CLUSTER_ON_IBM_CLOUD=true
+        ibmcloud login --apikey $IBMC_developer_APIKEY -a cloud.ibm.com -r eu-de -g "Developer-CI-and-QE"
+        ibmcloud ks cluster config --cluster $IBMC_K8S_CLUSTER_ID
+
+        # Integration tests
+        shout "| Running integration Tests on Kubernetes cluster in IBM Cloud"
+        make test-cmd-project
+        make test-integration-devfile
```

Comment:
So we're going to enable only these two test targets in the first pass? Rest to follow later?

Reply:
Yes, that is because those two test suites pass OK when I run them manually against the K8S cluster in the IBM Cloud, others have some failures. As agreed we are enabling only those that pass and then will add others as fixes become available.

Reply:
Looks like you removed `test-integration-devfile`. It isn't working well any more? 😞

Reply:
> Looks like you removed `test-integration-devfile`. It isn't working well any more? 😞

yes, some failures in `test-integration-devfile` again